.github/workflows/ci.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
         make ci
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GO_TEST_TIMEOUT_SCALE: 2
```

Comment:
Does it make sense to set this in the Makefile to prevent CI and local from drifting?